JitsiConference.js

```diff
@@ -2530,6 +2553,57 @@ JitsiConference.prototype._onIceConnectionEstablished = function(
 
 };
 
+/**
+ * Called when the chat room reads a new list of properties from jicofo's
+ * presence. The properties may have changed, but they don't have to.
+ *
+ * @param {Object} properties - The properties keyed by the property name
+ * ('key').
+ * @private
+ */
+JitsiConference.prototype._propertiesChanged = function(properties) {
+    const changed = isEqual(properties, this.properties);
+
+    this.properties = properties;
+    if (changed) {
+        this.eventEmitter.emit(JitsiConferenceEvents.PROPERTIES_CHANGED);
+
+        // Some of the properties need to be added to analytics events.
+        const analyticsKeys = [
+
+            // The number of jitsi-videobridge instances currently used for the
+            // conference.
+            'bridge-count',
+
+            // The conference creation time (set by jicofo).
+            'created-ms',
+            'octo-enabled'
+        ];
```

Comment:
You don't want to just log the whole properties object?

Reply:
I'd rather not, I don't know what we'll end up putting there (right now we use only the 3 properties mentioned here)